meteor/client/lib/editAttribute/edit-attribute-dropdown.tsx

```diff
@@ -0,0 +1,86 @@
+import { EditAttributeBase, IEditAttributeBaseProps, wrapEditAttribute } from './edit-attribute-base'
+import ClassNames from 'classnames'
+import * as React from 'react'
+
+export interface DropdownOption {
+	value: string
+	// Must be provided when 'useLabel' is true
+	label?: string
+}
+
+interface EditAttributeDropdownProps extends IEditAttributeBaseProps {
+	options: DropdownOption[]
+	useLabel?: boolean
```

Comment:
Is this needed? Can't this component display the label if and only if provided? Or does this mean it's using the label as a value? In that case why can't the options be provided in that fashion to the component?

Reply:
I think it was needed at some point before we decided to reduce the scope of my refactor.
I will take a second look at since it's a boolean flag and boolean flags are bad practices